content/cli/commands/cluster.md

```diff
@@ -1,5 +1,266 @@
 ---
-title: Fluvio Runner
-menu: Runner
-weight: 10
+title: Fluvio Cluster
+menu: Cluster
+weight: 50
 ---
+
+The `fluvio cluster` family of commands is used to install and manage your own
+Fluvio cluster. The two primary forms of self-hosted cluster are Kubernetes
+(via Minikube) and Local clusters, both of which can be automatically set up
```

Comment:
Local cluster needs an additional paragraph. `A local cluster runs the binary (non-containerized) images of an SC & SPU directly on your local machine. The local SPUs are referred to as `Custom SPUs` and they'll have port numbers of 5000 and beyond.`

Reply:
Fluvio uses Kubernetes for the internal metadata, hence it is required for both cluster types.